amplify/backend/api/colonycdapp/schema/schema.graphql

```diff
@@ -3751,6 +3751,8 @@ type ExpenditureMetadata @model {
   id: ID! # Self-managed, formatted as colonyId_nativeId
   fundFromDomainNativeId: Int!
   stages: [ExpenditureStage!]
+  expectedNumberOfPayouts: Int
+  expectedNumberOfTokens: Int
```

Comment:
Not marked as required so that old expenditures are unaffected. Uses of expectedNumberOfPayouts will fallback to expenditure.slots.length. Uses of expectedNumberOfTokens will fallback to a set of the payouts array mapped to include only the token address.